src/EX_browser.c

```diff
@@ -3465,7 +3465,7 @@ static info_filter_t* SB_InfoFilter_Parse(int* count)
 			// no regex is fine
 			filter->regex = pcre2_compile((PCRE2_SPTR)(split + 1), PCRE2_ZERO_TERMINATED, PCRE2_CASELESS, &error, &error_offset, NULL);
 			if (filter->regex == NULL) {
-				Con_Printf("Invalid rule definition: %s\n", error);
+				Con_Printf("Invalid rule definition: %d\n", error);
```

Comment:
This is a regression from how pcre1 was integrated. You should instead fetch the error message like in the gist from last night.

https://gist.github.com/dsvensson/8aa1202b8439718a53b04dcd82c24fec

Reply:
It gives an error code instead of a string. If someone wants to make it prettier, everyone is welcome.

The prints are very inconsistent in this code. Also the whole code in general does some weird if checking all over the place that seems unnecessary.

Reply:
Always leave code in better shape than you found it.